xml/http_cookies.xml

```diff
@@ -407,6 +407,15 @@
     <param pos="0" name="service.cpe23" value="cpe:/a:redmine:redmine:-"/>
   </fingerprint>
 
+  <fingerprint pattern="^syracuse.sid.\d+=.*">
+    <description>Sage X3 Syracuse Web Server</description>
+    <example>syracuse.sid.8124=8b102bf7-327c-4962-9279-550e72afcaa9; path=/; HttpOnly</example>
+    <param pos="1" name="cookie"/>
```

Comment:
This value should be set statically since it isn't part of a capture that could vary.